### PR TITLE
Kiali-1586 add a screenshot of Kiali 

### PR DESCRIPTION
**Pull request by vnugent**
** Describe the change **
Adding a static screenshot instead of hot linking to an image on kiali.io

** Issue reference **
Kiali-1586
** Backwards incompatible? **
No

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-core-pr-e2e-test [#139](https://paste.fedoraproject.org/paste/rpgzDGwkvJ0nX-6qCbALVw)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1512]](https://paste.fedoraproject.org/paste/eGetYZourLVup~YJ16OWQw)
  * :heavy_check_mark: Istio-Upstream [#[373]](https://paste.fedoraproject.org/paste/y62Qx20UtTstE7ILCyHrYg)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1572, 1573]](https://paste.fedoraproject.org/paste/S9ZnQYvVIErs4rE6RCjLpw)
  * :heavy_check_mark: install-bookinfo [#[882]](https://paste.fedoraproject.org/paste/PsHf~DG80hQBoWlUCacANA)
  * :heavy_check_mark: Kiali-on-Openshift [#[883]](https://paste.fedoraproject.org/paste/96-QXC4EcVwtIUEg069x4A)
  * :heavy_check_mark: run-kiali-e2e-tests [#[310]](https://paste.fedoraproject.org/paste/QbzhG9mrYKG0Tir7X5C7ow)


KIALI_NAME=kialicorepr544-istio-system

**Comment by vnugent** [comment]
@mwringe et al please review

File: README.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ image:https://img.shields.io/badge/license-Apache2-blue.svg["Apache 2.0 license"
 
 Kiali provides answers to the question _What are the microservices in my Istio service mesh doing?_
 
-image::https://www.kiali.io/images/features/feature3.png[Kiali Graph]
+image::kiali-full-ui.png[Kiali Graph, width=480]
```
Comment (abonas) [inline README.adoc]: Have you considered linking to the images in the kiali.io github repo? that would avoid duplicating and maintaining the image twice.
For instance linking here:
https://github.com/kiali/kiali.io/blob/master/static/images/gettingstarted/kiali-graph.png
or here:
https://github.com/kiali/kiali.io/blob/master/static/images/features/graph-overview.png

Comment (vnugent) [reply README.adoc]: That's how it was done in the past.  But when we made changes to kiali.io, we inadvertently broke the link.  As a result we have this current issue. Your thoughts?

Comment (abonas) [reply README.adoc]: if I understand correctly, before this PR, the linking was done directly to the image on the website itself (kiali.io). What I am proposing is linking to the image on github repo of kiali.io, not to the website itself.

Comment (vnugent) [reply README.adoc]: Ok I misunderstood. I can push an update with a link to graph-overview.png

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-core-pr-e2e-test [#156](https://paste.fedoraproject.org/paste/QtauMrnItYCCocJW4HBY6g)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1549]](https://paste.fedoraproject.org/paste/nkbgqrb7kquwUvvjBZnfEA)
  * :heavy_check_mark: Istio-Upstream [#[406]](https://paste.fedoraproject.org/paste/nNm8IVk4N4sQJjvQumynvA)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1652, 1653]](https://paste.fedoraproject.org/paste/MUTfN060A1lMs~CID93zlQ)
  * :heavy_check_mark: install-bookinfo [#[919]](https://paste.fedoraproject.org/paste/N~6tsbyHjbI7qB4xa3C4WA)
  * :heavy_check_mark: Kiali-on-Openshift [#[926]](https://paste.fedoraproject.org/paste/56zURaTiNEitEU7JZvc7qQ)
  * :heavy_check_mark: run-kiali-e2e-tests [#[347]](https://paste.fedoraproject.org/paste/7ZMZwbp7Uc~ySQ9KGiVwQA)


KIALI_NAME=kialicorepr544-istio-system

**Comment by vnugent** [comment]
I've updated the PR - using an image from kiali.io repo

**Pull request merged**